paltas/Configs/acs/config_train_noll.py

```diff
@@ -0,0 +1,3 @@
+from paltas.Configs.acs.config_train import *
```

Comment:
It would be good to either make the filename more descriptive or to add some comment about the intention (no lens light) in the a README / at the top of the file.